roles/cdi/tasks/deprovision.yml

```diff
@@ -1,46 +1,102 @@
 ---
 - name: Render {{ cdi_namespace }} ResourceQuota deprovision yaml
   template:
-    src:  cdi-resourcequota.yml
-    dest: /tmp/cdi-deprovision-resourcequota.yml
+    src: cdi-resourcequota.yml.j2
+    dest: "/tmp/cdi-deprovision-resourcequota.yml"
 
 - name: Delete {{ cdi_namespace }} ResourceQuota
   command: kubectl delete -f /tmp/cdi-deprovision-resourcequota.yml -n {{ cdi_namespace }} --ignore-not-found
 
```

Comment:
Need to cleanup the upload api.
```
oc delete apiservices v1alpha1.upload.cdi.kubevirt.io -n {{ cdi_namespace }}
```

Reply:
oh yeah, it is really annoying that garbage collection doesn't work for this in k8s 1.11.  Delete may not actually work, either.  Have you tried it @rthallisey?

Reply:
@mhenriks ya it worked when I removed it.

Reply:
done